meta-swupd/recipes-core/swupd-client/swupd-client/0001-Tolerate-quotes-in-os-release-files.patch

```diff
@@ -10,19 +10,18 @@ to the format defined in systemd.
 This patch removes quotes from the values before trying to
 transform them into integer version id.
 
-Signed-off-by: Dmitry Rozhkov <dmitry.rozhkov@intel.com>
-
-Upstream-Status: Backport (v3.0.0+)
+Upstream-Status: Accepted
```

Comment:
Why isn't backport right here? 

This patch was accepted upstream and we've backported it. By including the Backport tag and a version number we can more easily determine when this patch can be dropped.

http://www.openembedded.org/wiki/Commit_Patch_Message_Guidelines#Patch_Header_Recommendations